### PR TITLE
Fix for Welsh upload documents

### PR DESCRIPTION
**Pull request by adamdavies1**
https://dluhcdigital.atlassian.net/browse/FS-4032

### Change description
Adding fix for welsh documents not appearing in the areas that they should be

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Create a welsh application a check the areas where document uploads should be. All document upload section is a good example

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

File: api/routes/subcriterias/get_sub_criteria.py [hunk 1]
```diff
@@ -74,11 +74,22 @@ def get_themes_fields(
...
 def get_all_uploaded_document_field_answers(
-    fund_id: str, round_id: str
+    fund_id: str,
+    round_id: str,
+    language: str,
 ) -> list[dict]:
     sub_criterias = get_all_subcriteria(fund_id, round_id)
     filtered_answers = [
-        {**answer, "question": f"{theme['name']}, {answer['question']}"}
+        {
```
Comment (tferns) [inline api/routes/subcriterias/get_sub_criteria.py:83]: Is this the only place we pull back file uploads? I think on assessment frontend we may need to update some things to make sure these changes don’t break, from a quick look this endpoint only seems to be used for the “all uploaded documents” feature - what about regular sub criteria’s, did they work with this change?

**Review by tferns — APPROVED** [review]
Nice I like the way you’ve done this! Is there a corresponding PR on assessment frontend?

**Review by RamuniN — APPROVED** [review]
Great work 🔥. One question, as welsh language is handled in assessment backend no need to change anything in assessment frontend right?

**Comment by adamdavies1** [comment]
> Great work 🔥. One question, as welsh language is handled in assessment backend no need to change anything in assessment frontend right?

Yes, I had no need the change the frontend. I wanted to avoid it so the back end handles it all

**Pull request merged**